lib/handlers/get.js

```diff
@@ -93,8 +93,8 @@ function handler (req, res, next) {
     // Translate from the contentType found to the possibleRDFType desired
     translate(stream, baseUri, contentType, possibleRDFType, function (err, data) {
       if (err) {
-        debug('error translating: ' + req.originalUrl + ' ' + contentType + ' -> ' + possibleRDFType + ' -- ' + 500 + err.message)
-        return next(error(500, err.message))
+        debug('error translating: ' + req.originalUrl + ' ' + contentType + ' -> ' + possibleRDFType + ' -- ' + 406 + ' ' + err.message)
+        return next(error(406, err.message))
```

Comment:
this is not always true, since the error could be because of the translation issues, so this should be 500 and maybe before we check that if possibleRDFType is empty, then we return 406 (which I think we already do)